ARMDeploy/ohbooking.json

```diff
@@ -46,6 +46,7 @@
         "serviceBusTopicNotificationsName": "mailnotifications",
         "serviceBusTopicUpdatesName": "mailupdates",
         "serviceBusQueueNotificationResponsesName": "mailidresponses",
+        "serviceBusQueueBookingSlot": "newbookingslot",
```

Comment:
In that case I would make sure that the term ‘placeholder’ is in there somewhere. Maybe ‘mailPlaceholderUpdates’ ?